src/DIRAC/FrameworkSystem/DB/ProxyDB.py

```diff
@@ -395,6 +398,9 @@ def deleteProxy(self, userDN, userGroup=None, proxyProvider=None):
             return S_ERROR(", ".join(errMsgs))
         return result
 
+    _cache_getProxyStrength = TTLCache(maxsize=1000, ttl=600)
+
+    @cached(_cache_getProxyStrength, lock=Lock())
```

Comment:
I don't think there is any point in having a longer TTL than 10 minutes. It's already an eternity compared to the frequency with which this is called.

```suggestion
    @cached(TTLCache(maxsize=1000, ttl=600), lock=Lock())
```